### PR TITLE
Fix for VS build break and bug 7290513

### PR DESCRIPTION
**Pull request by dragonfly91**

### CODE REVIEW

File: src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Commands.RecoveryServices.Backup.Cmdlets.csproj [hunk 1]
```diff
@@ -43,7 +43,7 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\..\packages\Microsoft.Azure.Common.2.1.0\lib\net45\Microsoft.Azure.Common.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.Azure.Management.RecoveryServices.Backup, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+    <Reference Include="Microsoft.Azure.Management.RecoveryServices.Backup, Version=0.9.0.0, Culture=neutral, processorArchitecture=MSIL">
```
Comment (sudhakar4171) [inline src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Commands.RecoveryServices.Backup.Cmdlets.csproj]: why is this changed to 0.9.0.0?

File: src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Commands.RecoveryServices.Backup.Cmdlets.csproj [hunk 2]
```diff
@@ -106,7 +106,9 @@
     <Compile Include="RecoveryServicesBackupCmdletBase.cs" />
     <Compile Include="CmdletParameterHelpMessages.cs" />
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <Content Include="AzureRM.RecoveryServices.Backup.dll-help.xml" />
```
Comment (sudhakar4171) [inline src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Commands.RecoveryServices.Backup.Cmdlets.csproj]: why not give name like this - Microsoft.Azure.Commands.RecoveryServices.Backup.dll-help.xml?

File: src/ResourceManager/RecoveryServices/Commands.RecoveryServices/Microsoft.Azure.Commands.RecoveryServices.ARM.dll-help.xml [hunk 3]
```diff
@@ -1,189 +1,249 @@
 <?xml version="1.0" encoding="utf-8"?>
```
Comment (sudhakar4171) [inline src/ResourceManager/RecoveryServices/Commands.RecoveryServices/Microsoft.Azure.Commands.RecoveryServices.ARM.dll-help.xml]: what testing has been done to validate these help content?

File: src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Commands.RecoveryServices.Backup.Cmdlets.csproj [hunk 4]
```diff
@@ -106,7 +106,9 @@
     <Compile Include="RecoveryServicesBackupCmdletBase.cs" />
     <Compile Include="CmdletParameterHelpMessages.cs" />
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
```
Comment (sudhakar4171) [inline src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Commands.RecoveryServices.Backup.Cmdlets.csproj]: do we need to add this file name in .psd1 file?

File: src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Commands.RecoveryServices.Backup.Cmdlets.csproj (hunk 1 above)
Comment (dragonfly91) [inline src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Commands.RecoveryServices.Backup.Cmdlets.csproj]: This was auto generated when we added the reference

Comment (sudhakar4171) [inline src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Commands.RecoveryServices.Backup.Cmdlets.csproj]: All other dependencies are using 1.0 ..  to be consistent is it ok if we change this to 1.0 ?

File: src/ResourceManager/RecoveryServices/Commands.RecoveryServices/Microsoft.Azure.Commands.RecoveryServices.ARM.dll-help.xml (hunk 3 above)
Comment (dragonfly91) [inline src/ResourceManager/RecoveryServices/Commands.RecoveryServices/Microsoft.Azure.Commands.RecoveryServices.ARM.dll-help.xml]: I believe this testing has to be done by each individual scenario owners

File: src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Commands.RecoveryServices.Backup.Cmdlets.csproj (hunk 1 above)
Comment (dragonfly91) [inline src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Commands.RecoveryServices.Backup.Cmdlets.csproj]: I checked azure backup module and found that even there the version doesn't match exactly. Probably these are two different versions.

File: src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Commands.RecoveryServices.Backup.Cmdlets.csproj (hunk 4 above)
Comment (dragonfly91) [inline src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Commands.RecoveryServices.Backup.Cmdlets.csproj]: No
